<documentation title="Concatenation Spacing">
    <standard>
        <![CDATA[
The string concatenation token "." MUST be surrounded by spaces.
]]>
    </standard>
    <code_comparison>
        <code title="Valid: Using the string concatenation token with surrounding spaces.">
            <![CDATA[
$string = 'Hello'<em> </em>.<em> </em>'World!';
]]>
        </code>
        <code title="Invalid: Using the string concatenation token without preceding space.">
            <![CDATA[
$string = 'Hello<em>'.</em> 'World!';
]]>
        </code>
        <code title="Invalid: Using the string concatenation token without following space.">
            <![CDATA[
$string = 'Hello' <em>.'</em>World!';
]]>
        </code>
        <code title="Invalid: Using the string concatenation token without surrounding space.">
            <![CDATA[
$string = 'Hello<em>'.'</em>World!';
]]>
        </code>
    </code_comparison>
</documentation>
